Sources/Extensions/Watch/ComplicationController.swift

```diff
@@ -16,8 +16,8 @@ class ComplicationController: NSObject, CLKComplicationDataSource {
     // https://github.com/LoopKit/Loop/issues/816
     // https://crunchybagel.com/detecting-which-complication-was-tapped/
 
-    private func template(for complication: CLKComplication) -> CLKComplicationTemplate? {
-        Iconic.registerMaterialDesignIcons()
+    private func getComplicationModel(for complication: CLKComplication) -> WatchComplication? {
```

Comment:
prefixing functions with 'get' is not really a swift-ism that works super well -- apple's been leaning on doing this only for asynchronous returns (e.g. it provides a block)